src/main/java/com/qaprosoft/carina/demo/gui/components/AddedProductModal.java

```diff
@@ -0,0 +1,35 @@
+package com.qaprosoft.carina.demo.gui.components;
+
+import com.qaprosoft.carina.core.foundation.webdriver.decorator.ExtendedWebElement;
+import com.qaprosoft.carina.core.gui.AbstractUIObject;
+import com.qaprosoft.carina.demo.gui.pages.AutomationPracticeShoppingCartSummary;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+public class AddedProductModal extends AbstractUIObject {
+
+    @FindBy(xpath = ".//span[@class='product-name']")
```

Comment:
don't need a `.` in the beginning of xpath

Reply:
Ok, thought it was necessary to do like a relative path inside components. I will fix it